docs/core/integration-quality-scale/rules/iqs030.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "IQS030 - Errors are translatable"
+---
+
+IQS030 is as follows:
+
+Errors are translatable
+
+## Reasoning
+
+Sometimes something goes wrong and an error is shown to the user.
+Since Home Assistant is used by people all over the world, it is important that these errors are translatable.
+This increases the usability of Home Assistant for people who do not use the application in English.
```

Comment:
```suggestion
Since Home Assistant is used by people all over the world, it is important that these error messages are translatable.
This increases the usability of Home Assistant for people who do not use the application in English.

Home Assistant has builtin support for translating messages coming from the `HomeAssistantError` exception.
```